eslint.config.mjs

```diff
@@ -51,7 +51,6 @@ export default [
       'test/.tmp.*/**',
       'test/addons/??_*',
       'test/fixtures/**',
-      'test/message/esm_display_syntax_error.mjs',
       'tools/github_reporter/**',
       'tools/icu/**',
```

Comment:
While we're at it, it seems that this glob also doesn't have any failing files (yet).
```suggestion
```

Reply:
Yes, but those files exist. I'd rather just remove the non existent files, and look into that for a future PR